client/src/pages/courses/view.tsx

```diff
@@ -0,0 +1,150 @@
+import Checkbox from '@/src/shared/components/Checkbox';
+import Dropdown from '@/src/shared/components/Dropdown';
+import Radiobutton from '@/src/shared/components/Radiobutton';
+import SearchBar from '@/src/shared/components/SearchBar/SearchBar';
+import GridIcon from '@/src/shared/icons/GridIcon';
+import ListIcon from '@/src/shared/icons/ListIcon';
+import { LeftSideBar } from '@/src/shared/layouts/RightSideBar/LeftSideContainer';
+import { RightSideBar } from '@/src/shared/layouts/RightSideBar/RightSideBar';
+import Image from 'next/image';
+import { Fragment, useState } from 'react';
+import type { ReactNode } from 'react';
```

Comment:
- this is not necessary. But could be beneficial in our eyes if we have proper arrangements of imports.


```
// built-in packages

// shared components
// shared hooks
// page sections

// css
```

ex.

```
import { Fragment, useState } from 'react';
import Image from 'next/image';

import Checkbox from '@/src/shared/components/Checkbox';
import Dropdown from '@/src/shared/components/Dropdown';

import style from './home.module.css`
```